### PR TITLE
Update Guidelines for Submitting Lessons to ph-submissions

### LINKED ISSUE #1795: Preventing Build Errors on ph-submissions
The broad proposed action here is that we rethink how we ask authors to upload materials to the ph-submissions repository or otherwise think through the editorial workflow slightly differently. 

Ph-submissions has people push directly to it rather than make pull requests (presumably for ease of use by authors). But this means that if the site gets a build error the results are a little more hidden than they would be on our site. You get an email, but the email can be easy to ignore because it won't block the contribution from being part of the repo like it would in a pull request. Here’s an example screenshot of the commit log:

Screen Shot 2020-05-28 at 2 27 09 PM

There was an issue introduced several commits back that broke the site build, so it stopped rendering properly. The issue with this is that, until the build error is fixed, the build will continue to error until someone fixes it. And it won’t build on the subsequent pushes. That means, for example, that if I had had looked at the live, rendered version of my lesson in progress that I just committed to I might be wondering why changes I pushed weren’t showing up. The errors persist until someone notices them and fixes them, and there’s not really anyone tending to them. Most of the errors eventually get ironed out in the review process, but that can often take a while. They’re primarily errors introduced by authors who have made mistakes with the metadata or include syntax for figures. The build error here actually wasn’t resolved until  made changes to four different lessons in progress. No one is really tending the repo in that way, and there's not a good process for making sure that those sorts of errors don't take place. 

We discussed this a little on the technical team slack how we might avoid this sort of thing in the future. The consensus that I'd like to propose is the following:

- We tweak the editorial process to have the editor for a lesson post the initial file for the lesson and, in the process, make a first pass at making sure the metadata is formatted correctly. 
- We make a note about how the editorial/translation process should involve checking the commit log for build errors and show where to find them in the web interface (I don't think this is in the guidelines, but I might have missed it while re-reading just now). 

That would catch most of the errors I think and give a process for making sure they get caught regularly as part of the process.

### PR DESCRIPTION
**Pull request by walshbr**
Closes #1795. Note that this will be a draft pull request for feedback from the technical team and MEs. It will not be finalized until after the Portuguese team has finalized their work, as this pull request affects files that they are currently translating. 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [ ] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by walshbr** [comment]
@programminghistorian/technical-team - can one of you take a look at the instructions here to make sure that I've updated correctly based on our new policy to have authors email in their submissions for the editors to then upload the materials to ph-submissions.

@svmelton - since we just tried this out with the English team, could you read the diff as well to make sure it makes sense? https://github.com/programminghistorian/jekyll/pull/1844/files.

Note again that this is meant to remain drafted until after the Portuguese team is done translating their materials. So won't ask for review and translation until that time. @acrymble - I assume that leaving edits in draft form like this still respects their need to have the documents frozen so they remain in synch while also allowing us to keep working. But let me know if that's not the case.

**Comment by acrymble** [comment]
@walshbr if you're making changes the portuguese team might like the option of just doing it right the first time? Maybe send them a message and find out. Reducing overall effort is a good thing.

**Comment by DanielAlvesLABDH** [comment]
We will open and edit one file at a time, from the list mentioned in #1818, so if you want we can leave this file to last and then incorporate the changes made

**Comment by walshbr** [comment]
Yeah I agree @acrymble - I was thinking the same thing. the more I think about it, though, @DanielAlvesLABDH - I'm inclined to just hold this pull request for now until we get the Portuguese team entirely merged. It feels a little hairy to try to coordinate multiple versions and keep track of what files are in what working state. Even just trying to make sense of the plan in the abstract I am finding a little difficult. So I'll keep this drafted for now and we can touch base when PT is merged?

**Comment by DanielAlvesLABDH** [comment]
Fine

**Comment by svmelton** [comment]
Thanks, @walshbr—the diff looks good to me, whenever we're ready to move forward.

**Comment by walshbr** [comment]
@programminghistorian/technical-team is going to review the language here before asking for translations.

**Review by jenniferisasi — CHANGES_REQUESTED** [review]
Maybe on author-guidelines on step 1 you don't need to say that the account gives permission to upload; I think that might be confusing since they are not supposed to upload anything themselves. Maybe just "grant access to use our repository during he peer-review process". What do you think?

**Comment by walshbr** [comment]
@jenniferisasi I think under this system we would just be having the editors do the initial upload of new materials so they can do initial revisions to the metadata and syntax. But ultimately we want them to then start working from and revising that file, right? (we don't want the editor to have to continually get a new version and re-upload it for the authors). So maybe…

A clarifying note in step one as you say, clarifying that this first upload will be done by the editor. and then in step four or five noting that future revisions to the lesson should be done to the file on the ph submissions lesson itself. 

Does that seem to make sense to you as a process?

**Comment by jenniferisasi** [comment]
> @jenniferisasi I think under this system we would just be having the editors do the initial upload of new materials so they can do initial revisions to the metadata and syntax. But ultimately we want them to then start working from and revising that file, right? (we don't want the editor to have to continually get a new version and re-upload it for the authors). So maybe…
> 
> A clarifying note in step one as you say, clarifying that this first upload will be done by the editor. and then in step four or five noting that future revisions to the lesson should be done to the file on the ph submissions lesson itself.
> 
> Does that seem to make sense to you as a process?

It does! Send file to editor, editor uploads, then edits happen through GitHub. As it reads right now, the user could think: "oh, so I need to upload it and the editor has to find the file name in the repo".

**Comment by walshbr** [comment]
@jenniferisasi I made changes to reflect your suggestions if you want to modify/dismiss your review.

**Review by jenniferisasi — DISMISSED** [review]

**Comment by walshbr** [comment]
Just a note - I pinged the tech team on slack to make sure no one else had further comments before requesting translation.

**Comment by walshbr** [comment]
OK @programminghistorian/french-team @programminghistorian/spanish-team and @programminghistorian/portuguese-team. This is ready to translate. There are two files in question - 

en/author-guidelines.md
en/editor-guidelines.md

The diff to show which lines were changed can be found here - https://github.com/programminghistorian/jekyll/pull/1844/files. By default it will show a side-by-side view of the old file version on the left and the new file version on the right, and it should automatically show only the pieces of the files that were edited. Let me know if you have questions!

**Comment by jenniferisasi** [comment]
Working on the @programminghistorian/spanish-team version

**Comment by rivaquiroga** [comment]
> Working on the @programminghistorian/spanish-team version

Thanks! The changes to the Author Guidelines were already made on this PR: https://github.com/programminghistorian/jekyll/pull/1771. But we need to check if any changes were introduced after we updated that file.

**Comment by jenniferisasi** [comment]
> #1771

Oh! I forgot about that. So, yes, we have a couple of new change. Since this now involves me checking on 3 different files, I am going to save my translation and check it by the end of the week (can't do the 2 things now, translated and double check)

**Comment by spapastamkou** [comment]
Just to let you know that I started working on this and in the FR case it involves 3 files: authors, editors and translators guidelines. I find considerable differences in the authors guidelines (up to now) so I make further changes.

**Comment by spapastamkou** [comment]
Please @fdlaramee or @mariechristineb check changes in https://github.com/programminghistorian/jekyll/pull/1844/commits/24e7594e920ce1427d5a4b267ca902337b1ddc46 if able to (otherwise I'll eventually give review for the FR team)

**Comment by DanielAlvesLABDH** [comment]
Just to let you know that I updated the Portuguese files

**Review by DanielAlvesLABDH — DISMISSED** [review]
Made the changes to the PT files

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: je dirais "pourra se concentrer sur le fond, plutôt que sur la forme, afin de vous aider à [...]" (si ça ne s'éloigne pas trop de l'original)

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: "et qu'elle reste disponible"

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: je remplaçerais "guidance" par "des indications" ou "une ligne directrice"

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: Je dirais plutôt "Consultez nos [consignes aux évaluateurs et évaluatrices](/fr/consignes-evaluateurs) afin de vous familiariser avec notre processus de révision et de publication." (ou "si vous souhaitez vous familiariser")

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: "pour garantir" (sans le 'se')
"ou encore si vous considérez" 
j'enlèverais le 'enfin' et je remplacerais simplement par "ou pour quel autre souci que ce soit"
"afin que nous puissions"

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: "d'entrée - sentez-vous"

File: fr/consignes-auteurs.md
Comment (mariechristineb) [inline fr/consignes-auteurs.md]: Ailleurs sur le site, on n'utilise pas les titres

**Review by mariechristineb — COMMENTED** [review]
@spapastamkou I added comments to 24e7594

**Comment by rivaquiroga** [comment]
@jenniferisasi, el único cambio que faltaba en la guía para autores y la guía para traductores era el punto 5 de estas indicaciones. Lo demás ya lo habíamos traducido con Maria José (este PR estuvo un tiempo como draft, así que aprovechamos mientras tanto de incorporar estos cambios en #1771).
Ya incorporé en #1771 ese punto (y una frase extra que agregaron en el número 1), así que lo único que por parte del equipo de español faltaría en este PR es agregar esta nueva información a la Guía del Editor.

File: fr/consignes-auteurs.md
Comment (spapastamkou) [reply fr/consignes-auteurs.md]: j'ai tout intégré, merci pour la relecture.

**Comment by spapastamkou** [comment]
In integrated @mariechristineb comments in the FR guidelines and updated the branch.

**Comment by walshbr** [comment]
@jenniferisasi @DanielAlvesLABDH @spapastamkou - just a note that if the translations from your team are finished you should go ahead and give an approving review on the pull request so that I can merge. Thank you!

**Review by DanielAlvesLABDH — DISMISSED** [review]
PT translations are ok

**Review by spapastamkou — DISMISSED** [review]

**Comment by jenniferisasi** [comment]
@walshbr I probably have the silliest question ever: If I work directly on the branch for this PR, it doesn't show me the latest file that Riva pushed today (guía para autores) onto which I want to make the changes asked for in this PR ... if we update this branch somehow, will the right file show?

**Comment by walshbr** [comment]
It's not a silly question @jenniferisasi - you have to re-merge the main branch into older branches when the main one changes. And there's no clear way to do that in the github interface that I've ever seen (@ZoeLeBlanc might know otherwise). I just did so, though, so you should see the latest files now.

**Comment by jenniferisasi** [comment]
Ok, I think with my uploads this issue is ready to be closed (once we re-approve).

**Comment by walshbr** [comment]
I think if you just approve @jenniferisasi I should be good to merge. Nothing has changed on the Fr or Pt sides that would require them to re-review.

**Review by jenniferisasi — APPROVED** [review]
approved!

**Pull request merged**
